app.py

```diff
@@ -22,13 +22,21 @@
 
 def send_email(subject, body, recipients=None):
     """send an email to the selected recipients with a given body context."""
+
+    # TODO: hack for now, validation should happen when program starts
+    email_from = os.getenv("EMAIL_FROM")
+    if not email_from:
```

Comment:
I would prefer using a default value then raising an exception

Reply:
What is a default value for this?

Reply:
Maybe; batyam@redhat.com 
We need to verify that batyam ML exists.

Reply:
and even if write this.. what about this project being open source and generic?
should the default be an email@redhat.com or any email in particular?

Reply:
good point :-)